etc/services/Step 1 - Collect.cfg

```diff
@@ -2,58 +2,59 @@
tion\":1,\"target\":\"CollectRecent\",\"left\":0,\"top\":0,\"width\":500,\"align\":\"left\",\"content\":\"#####Step 2\\nHere we convert the raw data into a sliding window list. A sliding window list will remove the oldest data point in the list to make room for a new data point.  This is done to ensure a consistent data shape for plotting charts on the UI.\\n\\n\\n\"},{\"position\":1,\"target\":\"Simplify\",\"left\":0,\"top\":0,\"width\":500,\"align\":\"left\",\"content\":\"#####Step 3 \\nUsing the list from the SlidingWindow block, this block reduces the overall number of data points in the list while still maintaining the same shape of the plot. It is mainly housekeeping to make the chart on the UI look better.\\n\\n\\n\"},{\"position\":1,\"target\":\"JoinFreeDraw\",\"left\":0,\"top\":0,\"width\":500,\"align\":\"left\",\"content\":\"#####Step 4\\nThis groups all of the incoming signals into one signal prior to being published to the UI. \\n\"},{\"position\":1,\"target\":\"SendFreeDrawUI\",\"left\":0,\"top\":0,\"width\":500,\"align\":\"left\",\"content\":\"#####Step 5 \\nThe publisher block allows you to send data to any participants in the nio system. This could be another service, a separate instance, or a UI. This block sends the consolidated accelerometer data to the UI over the `mobiledemo.freedraw` topic.\\n\\n\\n\"},{\"position\":0,\"target\":null,\"left\":0,\"top\":0,\"width\":800,\"align\":\"left\",\"content\":\"##Tutorial Introduction\\nWelcome to the *tutorial: shake your phone*. As a part of this tutorial, we will connect your smartphone to a nio system that contextualizes and visualizes data coming from your phone\u2019s sensors. Let\u2019s get started!\\n\\nRequirements: Smartphone, Computer\\n\\n    -  On your smartphone, navigate to https://b.n.io/mobile\\n    -  Sign in to your nio account\\n    -  Select your mobile tutorial system\u2019s name \\n    -  On your desktop browser, navigate to https://b.n.io/mobile in an new tab \\n    -  Select your mobile tutorial system\u2019s name \\n    -  Continue the tutorial by following along the service below!\\n\\n\\n\"},{\"position\":0,\"target\":null,\"left\":100,\"top\":1300,\"width\":600,\"align\":\"center\",\"content\":\"The above service works a little nio magic, but the output is still messy to the human eye. In the next two services, we explore different, more useful transformations of your phone\u2019s sensor data and teach basic nio service building concepts. Select *Step 2 - Contextualize* in the service panel to continue...\\n\"},{\"position\":0,\"target\":null,\"left\":0,\"top\":300,\"width\":800,\"align\":\"center\",\"content\":\"###Service Overview\\nThis first service subscribes to the sensor data coming off your phone. The service is fully configured. **All you have to do is start it, by clicking on the start icon located in the contextual toolbar above the canvas.** Once it's running, go check out the desktop UI, and see what happens when you move your phone. \\n\"}]}",
```

Comment:
I'm not sure that this change is needed, it would be best to exclude this from your commit. Let me know if you need a hand cleaning it up.